docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+---
+description: How to choose and connect to a testnet
+---
+
+# Connect to a Testnet
+
+The PegaSys Teku client allows you to connect a beacon-chain node to a public testnet and run validators on these networks. There are different ETH2 testnets to choose from.
+
+!!! note
+
+    Due to the fast pace of client and testnet development, this guide recommends connecting to the [Schlesi Testnet](https://github.com/goerli/schlesi). However, networks can experience stability issues and are prone to regular resets. It's recommended to regularly check the network's and client's documentation for updates.
+
+**Prerequisites**:
+
+* [Install Teku](/HowTo/Get-Started/Install-Binaries) Ethereum 2.0 client or [Build Teku from Source](/HowTo/Get-Started/Build-From-Source).
+
+Teku allows to select preconfigured networks with the `--network` flag:
+
+```bash
```

Comment:
Could you try to use the same CLI syntax layout than in https://github.com/PegaSysEng/doc.teku/blob/master/docs/Reference/CLI/CLI-Syntax.md ?

It may be something like:

```
    ```bash tab="Syntax"
    -n, --network=<NETWORK>
    ```
    
    ```bash tab="Command Line"
    --network=mainnet
    ```
    
    ```bash tab="Environment Variable"
    TEKU_network=mainnet
    ```
    
    Represents which network to use.
    The default is `minimal`.
```

Reply:
Done that but it completely bricks the rendered version.

![Screenshot_2020-05-07 Connect to a Testnet - Teku](https://user-images.githubusercontent.com/58883403/81289402-a7881a00-9066-11ea-9c50-8a8c80284359.png)

Reply:
Hum yes if the superfences is not activated... Let me try to build your PR locally. I also sent you a ping on Gitter if you have some time to discuss this.

Reply:
Tested locally and it works well for me so I think it's just a small conflict in Python packaging, we already saw that and it's not very hard to fix but requires some patience...
I can help you with that. The thing for me to take away from this is that I have to document the doc building process and doc dev environment a bit more. It's not the same product, but I'm not going to lie, it's written by me too, so if you would like more details on the doc building, you can have a look at https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation
Except that Besu doesn't use the common Git submodule, it's mostly the same thing, I try to be consistant on the tooling. I also sent you a message on Gitter if you want to chat more interactively.